### PR TITLE
opencv 4.5.4

### PR DESCRIPTION
**Pull request by cclauss**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

Recapitulates #87064

~Removed `openalpr` because it has not been updated since 2016 and fails tests on a well-documented `AssertionError`:~
* ~https://github.com/openalpr/openalpr/search?q=is_loaded&type=issues~

### CODE REVIEW

**Review by carlocab — COMMENTED** [review]
```
  libc++abi: terminating with uncaught exception of type std::bad_alloc: std::bad_alloc
```
Deleting `openalpr` to deal with the system not being able to allocate memory seems like the wrong thing to do here.

**Comment by cclauss** [comment]
Please add the [`python-3.10-migration` label](https://github.com/Homebrew/homebrew-core/labels/python-3.10-migration) to this PR because this `openalpr` problem blocks:
* #89948 (this PR)
* #85282
* #87287

**Comment by cclauss** [comment]
@matthill Do you have any insight into these failures we gt when trying to test against:
https://upload.wikimedia.org/wikipedia/commons/b/b0/California_2018_license_plate_%28USA%29.jpg
```
==> brew test --retry --verbose openalpr
==> FAILED
Full test openalpr output
  ==> Testing openalpr
  cp -p /Users/brew/Library/Caches/Homebrew/downloads/195774c820111f890a628e11f0544ab624568a4435ed3e29408b0a5e3c18a70d--California_2018_license_plate_(USA).jpg /private/tmp/openalpr--testdata-20211125-79036-1lrsu4r/California_2018_license_plate_(USA).jpg
  ==> /opt/homebrew/Cellar/openalpr/2.3.0_2/bin/alpr plate.jpg
  libc++abi: terminating with uncaught exception of type std::bad_alloc: std::bad_alloc
  
  Error: openalpr: failed
  An exception occurred within a child process:
    Minitest::Assertion: Expected: 0
    Actual: nil
```

**Comment by bayandin** [comment]
> @matthill @bayandin Do you have any insight into these failures we get when trying to test against: https://upload.wikimedia.org/wikipedia/commons/b/b0/California_2018_license_plate_%28USA%29.jpg
> 
> ```
> ==> brew test --retry --verbose openalpr
> ==> FAILED
> Full test openalpr output
>   ==> Testing openalpr
>   cp -p /Users/brew/Library/Caches/Homebrew/downloads/195774c820111f890a628e11f0544ab624568a4435ed3e29408b0a5e3c18a70d--California_2018_license_plate_(USA).jpg /private/tmp/openalpr--testdata-20211125-79036-1lrsu4r/California_2018_license_plate_(USA).jpg
>   ==> /opt/homebrew/Cellar/openalpr/2.3.0_2/bin/alpr plate.jpg
>   libc++abi: terminating with uncaught exception of type std::bad_alloc: std::bad_alloc
>   
>   Error: openalpr: failed
>   An exception occurred within a child process:
>     Minitest::Assertion: Expected: 0
>     Actual: nil
> ```

If I understand it correctly, `openalpr` can't allocate enough memory for processing that image. You could try to use a smaller file: https://upload.wikimedia.org/wikipedia/commons/thumb/b/b0/California_2018_license_plate_%28USA%29.jpg/320px-California_2018_license_plate_%28USA%29.jpg

**Comment by cclauss** [comment]
This PR now reverts b005c357c3846dd6f380e4e2e61d305f007429fd

File: Formula/openalpr.rb [hunk 1]
```diff
@@ -4,101 +4,35 @@ class Openalpr < Formula
...
 
   bottle do
-    sha256 arm64_big_sur: "4413a608d96584a63180fcb7a8d1c794ef8277010b9bd76f03c3bdfd60e3fe5a"
-    sha256 big_sur:       "a686b58100f9c397a2be1eb70595c773440e26e652a8539745bcfebd38359b12"
-    sha256 catalina:      "c1ba59a1d018a65f019ea162dd44efefc28ef54720f5b702c3763f21b5bdbb65"
-    sha256 mojave:        "6b61f23a1832eaea3acebf2ac5333d96eaa2ac9f978b8c1dbe6cfabadf0e980c"
-    sha256 x86_64_linux:  "a37d2f029a47097ea7c81936ed0400451780ae75594cf4aae10cc1647e5bc93e"
+    sha256 big_sur:  "32953a90e352cc25eea99daad7fac67c7ce11f042fe366af029b16cc3f372c07"
+    sha256 catalina: "0a05746932c63e1d0dd1749a7213ed0a9cff7d7da2cdfe49e032aeb1b1da56ad"
+    sha256 mojave:   "64dd8cef3f2fe311304e0fffae5ed50dd37123d6611aae751ba272be956bf334"
```
Comment (carlocab) [inline Formula/openalpr.rb:12]: Please don't touch the bottle block.

**Review by carlocab — CHANGES_REQUESTED** [review]

**Pull request closed without merging**